tracer/transport.go

```diff
@@ -22,6 +25,24 @@ type Transport interface {
 	SetHeader(key, value string)
 }
 
+// NewHTTPTransport returns a new Transport implementation that sends traces to
+// a trace agent running on the given hostname and port. If the zero values for
+// hostname and port are provided, the default values will be used ("localhost"
+// for hostname, and "7777" for port).
+//
+// In general, using this method is only necessary if you have a trace agent
+// running on a non-default port or if it's located on another machine.
+func NewHTTPTransport(hostname, port string) Transport {
+	if hostname == "" {
+		hostname = defaultHostname
+	}
+	if port == "" {
+		port = defaultPort
+	}
+	url := fmt.Sprintf("http://%s:%s/v0.3/traces", hostname, port)
```

Comment:
Good catch. It'd probably make sense to change `httpTransport` to have a `legacyURL` as well. Will fix.